### PR TITLE
1.9 Final release announcement

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: _posts/2022-04-06-debezium-1.9-final-released.adoc [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+i.e. also if you'd capture changes from only one single logical database,
+you'll be using the multi-partition mode then.
+Multi-partition mode will also be rolled out for other connectors where it's possible, e.g. for the connectors for Oracle and IBM Db2.
+
+Thanks a lot to Sergei and team for their excellent collaboration around that feature!
+
+== Further Changes
+
+Let's take a look at some more features new in Debezium 1.9.
+First, Debezium Server has now a sink adaptor for HTTP,
```
Comment (Naros) [inline _posts/2022-04-06-debezium-1.9-final-released.adoc]:
```suggestion
First, Debezium Server now includes a sink adaptor for HTTP,
```

File: _posts/2022-04-06-debezium-1.9-final-released.adoc [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+which means it can be used as a "native" event source for Knative Serving,
+without the need for sending messages through a message broker like Apache Kafka first.
+
+Then, the friendly folks over at Redis Labs stepped up and contributed several improvements to how Debezium (Server) integrates with Redis Streams: besides several performance improvements,
+the database history for connectors like the MySQL one can now be stored in Redis,
+also offsets which can be stored there now.
+But they didn't stop there: for instance, Debezium Server now supports custom configuration providers,
+as already known in Kafka Connect.
```
Comment (Naros) [inline _posts/2022-04-06-debezium-1.9-final-released.adoc]:
```suggestion
as already provided in Kafka Connect.
```

File: _posts/2022-04-06-debezium-1.9-final-released.adoc [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+Going forward, the Redis Labs teams is planning on further cool improvements to Debezium at large,
```
Comment (Naros) [inline _posts/2022-04-06-debezium-1.9-final-released.adoc]:
```suggestion
Going forward, the Redis Labs teams are planning on further cool improvements to Debezium at large,
```
or
```suggestion
Going forward, the Redis Labs team is planning on further cool improvements to Debezium at large,
```
I'm not sure if "team" should be plural or not, but that influences the verb form that follows.

File: _posts/2022-04-06-debezium-1.9-final-released.adoc [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+* Use Java 11 as a baseline (for instance allowing to emit JDK Flight Recorder events for better diagnostics)
+* Default to multi-partition mode metrics (more consistent)
+* Make default topic names more consistent (for instance for the heartbeat topic)
+* Change the default type mappings for a small number of column types
+
+Planning for this is in full swing right now,
+and you are very much invited to join the https://groups.google.com/g/debezium/[discussion] either on the mailing list or on the https://issues.redhat.com/browse/DBZ-3899[DBZ-3899] issue in Jira.
+Note that while we want to take the opportunity to clean up some odditities which have accumulated over time,
+backwards compatibility will be key concern as always, and we'll try to minimize the impact on existing users as much as possible.
+But as you would expect it from a new major release, upgrading my take a slightly larger effort in comparison to the usual minor releases.
```
Comment (Naros) [inline _posts/2022-04-06-debezium-1.9-final-released.adoc]:
```suggestion
But as you would expect it from a new major release, upgrading may take a slightly larger effort in comparison to the usual minor releases.
```

**Review by Naros — COMMENTED** [review]
Just some minor nit-picks :)

File: _posts/2022-04-06-debezium-1.9-final-released.adoc (hunk 3 above)
Comment (gunnarmorling) [reply _posts/2022-04-06-debezium-1.9-final-released.adoc]: Yeah, "team" is what I meant to say. Thx!

**Comment by gunnarmorling** [comment]
Thx for reviewing, @Naros!

**Pull request merged**